addons/game.controller.snes/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.snes"
+        name="SNES Controller"
+        version="1.0.6"
+        provider-name="Team Kodi">
+    <requires>
+    </requires>
```

Comment:
as it is empty no need to list it

Reply:
This empty tag appears in all controllers in https://github.com/xbmc/repo-resources/tree/leia. Do you have a script that removes this?

Reply:
Yes, will create a PR.

Reply:
SNES is now in repo-resources and master. So is 360. Should we remove SNES from repo-resources, or should we add 360 to repo-resources?

Reply:
NVM saw game.controller.default is already in repo-resources